### PR TITLE
Update jBox

### PR DESCRIPTION
**Pull request by Dkamps18**
# Pull request template for new library

Pull request for issue: none
Related issue(s): none

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/StephanWagner/jBox
* Official website (optional, not the repository): https://stephanwagner.me/jBox
* NPM package url (optional):
* License and its reference: MIT
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 92
  * Count of stars: 1.1k
  * Count of forks: 262
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week: 462
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [ ] This lib was not found on cdnjs repo
* [ ] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [ x Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by Dkamps18** [comment]
That are all the missing versions. 
(Sorry for the slight spam because of the repo size i'm using github's web ui to upload which limits me to 100 files)

**Comment by MattIPv4** [comment]
All good, we all use the web ui :)

File: ajax/libs/jBox/package.json [hunk 1]
```diff
@@ -28,10 +28,11 @@
     "target": "git://github.com/StephanWagner/jBox.git",
     "fileMap": [
       {
-        "basePath": "Source",
+        "basePath": "dist",
         "files": [
           "jBox*",
-          "themes/*.css"
+          "themes/*.css",
+          "plugins/*"
```
Comment (MattIPv4) [inline ajax/libs/jBox/package.json]:
```suggestion
          "plugins/*.(css|js)"
```
This should cover everything needed whilst being clearer?

**Comment by Dkamps18** [comment]
Added, I forgot you could do it like that :|

**Comment by Dkamps18** [comment]
New PR: https://github.com/cdnjs/cdnjs/pull/13756

**Pull request closed without merging**